statistics/most_attended_competitions_in_single_week.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "../core/statistic"
+
+class MostAttendedCompetitionsInSingleWeek < Statistic
+  def initialize
+    @title = "Most attended competitions in a single week"
+    @table_header = { "Competitions" => :right, "Person" => :left, "&nbsp;Start&nbsp;date&nbsp;" => :left, "&nbsp;End&nbsp;date&nbsp;&nbsp;&nbsp;" => :left, "Year" => :left, "List" => :left }
+  end
+
+  def query
+    <<-SQL
+      SELECT
+        attended_within_week,
+        CONCAT('[', person.name, '](https://www.worldcubeassociation.org/persons/', person.id, ')') person_link,
+        week_start_date,
+        week_end_date
+        competitions_year,
+        competition_links
+      FROM (
+        SELECT
+          COUNT(*) attended_within_week,
+          personId,
+          DATE_ADD(competition.start_date, INTERVAL(-1-WEEKDAY(competition.start_date)) DAY) week_start_date, 
+          DATE_ADD(competition.start_date, INTERVAL(5-WEEKDAY(competition.start_date)) DAY) week_end_date,
```

Comment:
I think it makes more sense to count weeks as Monday-to-Sunday, because this way we capture a single weekend (also in line with the other stat)

```suggestion
          DATE_ADD(competition.start_date, INTERVAL(-WEEKDAY(competition.start_date)) DAY) week_start_date, 
          DATE_ADD(competition.start_date, INTERVAL(6 - WEEKDAY(competition.start_date)) DAY) week_end_date,
```